ui/src/app/applications/components/utils.tsx

```diff
@@ -1471,3 +1471,10 @@ export const userMsgsList: {[key: string]: string} = {
     groupNodes: `Since the number of pods has surpassed the threshold pod count of 15, you will now be switched to the group node view.
                  If you prefer the tree view, you can simply click on the Group Nodes toolbar button to deselect the current view.`
 };
+
+export function getAppUrl(app: appModels.Application) {
```

Comment:
```suggestion
export function getAppUrl(app: appModels.Application): string {
```

Reply:
Updated.

Reply:
@andrii-korotkov-verkada, could you please add a return type?

Reply:
It's added in a new version of the PR.